Tests/UnitTests/Shared/Shared.cpp

```diff
@@ -11,9 +11,26 @@
 #include <gtest/gtest.h>
 #include <future>
 #include <iostream>
+#ifdef __ANDROID__
+#include <AndroidExtensions/StdoutLogger.h>
+#endif
 
 namespace
 {
+#ifdef __ANDROID__
+    // Global flag to track if StdoutLogger has been initialized
+    static bool s_stdoutLoggerInitialized = false;
+
+    void EnsureStdoutLoggerStarted()
+    {
+        if (!s_stdoutLoggerInitialized)
+        {
+            android::StdoutLogger::Start();
+            s_stdoutLoggerInitialized = true;
+        }
+    }
+#endif
```

Comment:
I don't understand this. It looks like this logic was moved from JNI.cpp to here. Why? This is a shared file. It is not intended to have platform specific logic.

Reply:
draft PR is up in AndroidExtensions. iniital integration into this PR branch looks good. https://github.com/BabylonJS/AndroidExtensions/pull/15